src/astarte-charts/dataKinds.ts

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
we're starting to abuse this a little bit too much.
We could consider disabling it entirely or split our files acordingly.

Reply:
As of now, it makes sense to keep the rule and split classes into multiple files, since we could add more and more dataKinds in the future thus polluting this file.